modules/distr-tracing-install-otel-operator.adoc

```diff
@@ -0,0 +1,51 @@
+////
+This module included in the following assemblies:
+- distr_tracing_install/distr-tracing-installing.adoc
+////
+
+[id="distr-tracing-otel-operator-install_{context}"]
+= Installing the {OTELName} Operator
+
+#TECH PREVIEW BOILERPLATE HERE#
```

Comment:
Just noting that here and previously you have a note to add the tech preview module or similar

Reply:
Yeah, I hadn't meant to upload this file, it's still a draft.  But it's not linked anywhere so I decided to just leave it for now.

Reply:
But at least this way I'm making sure all the edits make it into this file.  *Silver lining*